testgres/node.py

```diff
@@ -989,13 +990,17 @@ def safe_psql(self, query=None, expect_error=False, **kwargs):
 
         # force this setting
         kwargs['ON_ERROR_STOP'] = 1
-
-        ret, out, err = self.psql(query=query, **kwargs)
+        try:
+            ret, out, err = self.psql(query=query, **kwargs)
+        except ExecUtilException as e:
+            ret = e.exit_code
```

Comment:
seems it brings bad condition to ```if ret:``` below, isn't it? still not sure

Reply:
It is checking in test - test_psql, in the last line. ret becoming equals 1 and throwing QueryException